meinberlin/apps/dashboard2/utils.py

```diff
@@ -0,0 +1,134 @@
+from functools import lru_cache
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+from . import components
+
+
+def get_project_dashboard(project):
```

Comment:
Not sure that utils is a good place for this dashboard interface. Maybe just move them to `__init__.py`?